wishlist/app/com/commercetools/sunrise/wishlist/controllers/SunriseWishlistController.java

```diff
@@ -0,0 +1,55 @@
+package com.commercetools.sunrise.wishlist.controllers;
+
+import com.commercetools.sunrise.framework.controllers.SunriseContentController;
+import com.commercetools.sunrise.framework.controllers.WithQueryFlow;
+import com.commercetools.sunrise.framework.hooks.EnableHooks;
+import com.commercetools.sunrise.framework.reverserouters.SunriseRoute;
+import com.commercetools.sunrise.framework.reverserouters.wishlist.WishlistReverseRouter;
+import com.commercetools.sunrise.framework.template.engine.ContentRenderer;
+import com.commercetools.sunrise.framework.viewmodels.content.PageContent;
+import com.commercetools.sunrise.wishlist.WishlistFinder;
+import com.commercetools.sunrise.wishlist.WithRequiredWishlist;
+import com.commercetools.sunrise.wishlist.viewmodels.WishlistPageContentFactory;
+import io.sphere.sdk.shoppinglists.ShoppingList;
+import play.mvc.Result;
+
+import javax.inject.Inject;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * This controller is used to view the current wishlist.
+ */
+public class SunriseWishlistController extends SunriseContentController implements WithQueryFlow<ShoppingList>, WithRequiredWishlist {
```

Comment:
It should be `abstract` (even though it has no abstract method left to be implemented, it won't work by default without an associated templateName, so in general it should not be used directly).